staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go

```diff
@@ -166,12 +170,13 @@ func WithLatencyTrackers(parent context.Context) context.Context {
 // an instance of LatencyTrackers is added. Tracers use given clock.
 func WithLatencyTrackersAndCustomClock(parent context.Context, c clock.Clock) context.Context {
 	return WithValue(parent, latencyTrackersKey, &LatencyTrackers{
-		MutatingWebhookTracker:   newSumLatencyTracker(c),
-		ValidatingWebhookTracker: newMaxLatencyTracker(c),
-		StorageTracker:           newSumLatencyTracker(c),
-		TransformTracker:         newSumLatencyTracker(c),
-		SerializationTracker:     newSumLatencyTracker(c),
-		ResponseWriteTracker:     newSumLatencyTracker(c),
+		MutatingWebhookTracker:     newSumLatencyTracker(c),
+		ValidatingWebhookTracker:   newMaxLatencyTracker(c),
+		PriorityAndFairnessTracker: newMaxLatencyTracker(c),
```

Comment:
(asuming either max or sum works since we expect at most 1 queue item per request)